app/assets/stylesheets/home.scss

```diff
@@ -1,9 +1,41 @@
 body {
-  background: image_url("polygon_green.jpg") no-repeat center center fixed;
+  background: image_url("gray_gradient.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
```

Comment:
Properties should be ordered background, -moz-background-size, -o-background-size, -webkit-background-size, background-size